syntax = "proto3";

package ImageService;

service NewImageService {
  rpc NewImage (ImageRequest) returns (ImageReply) {}
  rpc TestConnection (Test) returns (Test) {}
}

// The request message containing the user's name.
message ImageRequest {
    bytes data =1;
    int32 height =2;
    int32 width =3;
    int32 type =4;
}

// The response message containing the greetings
message ImageReply {
  string message = 1;
}

message Test{
  string message = 1;
}
